### PR TITLE
Add Team Rename command

### PR DESCRIPTION
**Pull request by drgarcia1986**
- Update database (teams table)
- Update Namespaces labels (`teresa.io/team`)

---
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa/366)

### CODE REVIEW

File: pkg/client/cmd/team.go [hunk 1]
```diff
@@ -196,3 +208,34 @@ func teamRemoveUser(cmd *cobra.Command, args []string) {
...
+	conn, err := connection.New(cfgFile, cfgCluster)
+	if err != nil {
+		client.PrintErrorAndExit("Error connecting to server: %v", err)
+	}
+	defer conn.Close()
+
+	cli := teampb.NewTeamClient(conn)
+	req := &teampb.RenameRequest{OldName: oldTeam, NewName: newTeam}
+	if _, err := cli.Rename(context.Background(), req); err != nil {
+		client.PrintErrorAndExit("Error connecting to server: %v", err)
```
Comment (aguerra) [inline pkg/client/cmd/team.go]: I guess this should be `client.PrintErrorAndExit(client.GetErrorMsg(err))`, right?

File: pkg/server/app/fake_test.go [hunk 2]
```diff
@@ -182,6 +182,30 @@ func TestFakeOperationsList(t *testing.T) {
...
+		{&App{Name: "teresa", Team: "vimmers"}, "gophers", 0},
+	}
+
+	fake := NewFakeOperations()
+
+	for _, tc := range testCases {
+		fake.(*FakeOperations).Storage[tc.app.Name] = tc.app
+		apps, err := fake.ListByTeam(tc.teamName)
+		if err != nil {
+			t.Fatal("error getting app list by team:", err)
```
Comment (aguerra) [inline pkg/server/app/fake_test.go]: Space after team

File: pkg/server/team/handlers_test.go [hunk 3]
```diff
@@ -297,3 +297,54 @@ func TestRemoveUserPermissionDenied(t *testing.T) {
...
+	s := NewService(fake)
+	ctx := context.WithValue(context.Background(), "user", &database.User{Email: "gopher", IsAdmin: true})
+
+	newName := "gophers"
+	req := &teampb.RenameRequest{OldName: oldName, NewName: newName}
+	if _, err := s.Rename(ctx, req); err != nil {
+		t.Fatal("Got error renaming team:", err)
+	}
+
+	oldTeam := fake.(*FakeOperations).Storage[newName]
```
Comment (aguerra) [inline pkg/server/team/handlers_test.go]: oldTeam -> newTeam ?

File: pkg/server/app/fake_test.go (hunk 2 above)
Comment (drgarcia1986) [reply pkg/server/app/fake_test.go]: Actually isn't necessary, go appends a space after the main string in non-f (printf vs print) functions.

File: pkg/server/team/handlers_test.go (hunk 3 above)
Comment (drgarcia1986) [reply pkg/server/team/handlers_test.go]: no, the team is the same, that method only changes the name.

File: pkg/client/cmd/team.go (hunk 1 above)
Comment (drgarcia1986) [reply pkg/client/cmd/team.go]: 🤦‍♂️ you're right!

**Comment by drgarcia1986** [comment]
@aguerra updated

**Comment by drgarcia1986** [comment]
> I saw you using if `err =` instead of `if err :=`, is it on purpose?

To tell the truth, I made this PR without time 😞, I'll change

**Comment by drgarcia1986** [comment]
I ran an `ag "if err =" --ignore vendor` and found this pattern in several files 🤔

**Comment by drgarcia1986** [comment]
@yagonobre 
> would be nice if you add a comment to all exported things
Really ? 😐

**Comment by yagonobre** [comment]
@drgarcia1986 not mandatory, maybe in another time

**Pull request merged**